server/plugins/Gotify.json

```diff
@@ -8,19 +8,9 @@
     {
       "name": "URL",
       "label": "Gotify URL (Without http/s)",
+      "description": "Gotify domain URL eq. https://example.org (Note: without /message)",
```

Comment:
These two lines contradict - should it include http/s or not? Also will this impact people with things already in those settings?

Reply:
Oh, right, I forgot to fix this label. I will fix it.

Reply:
@davidmckenzie I fixed label and added some compatibility checks, so it will work in most cases.